_docs/howto/redaxo_4_5_upgrade.md

```diff
@@ -62,9 +89,13 @@ Folgende Schritte sind notwendig um alte Redaxo 4 Passwörter auch in Redaxo 5 v
 $users = rex_sql::factory()
     ->setDebug(true)
     ->setTable(rex::getTable('user'))
-    // ->setWhere() //nur redaxo 4 accounts
+     //Redaxo 4 Redakteur aus dem Beispiel wurde mit der Id 2 in die Tabelle rex_user importiert
+     //natürlich sind auch mehrere Datensätze gleichzeitig selektierbar, es dürfen allerdings nur die importierten Redaxo 4 User sein!
+    ->setWhere(['id' => 2])
     ->select();
-
+```
+Sobald `$users` alle Redaxo 4 Redakteure enthält, kann folgendes Script in einem Modul oder Template **einmalig** ausgeführt werden. Alte Redaxo 4 Passwörter werden dadurch umgewandelt-
```

Comment:
Alternative Idee: Würde ich behelfsweise in einem REDAXO-PHP-Cronjob ablegen und ausführen

Reply:
Würde ich tendenziell weiterhin in Template / Module laufen lassen, einfach auf Grund der Aufwendigkeit.